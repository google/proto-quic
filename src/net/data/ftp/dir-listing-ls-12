total 14
lrwxrwxrwx   1 other          7 Sep  1  2005 bin -> usr/bin
dr-xr-xr-x   2 other        512 Aug  9  2004 dev
dr-xr-xr-x   2 other        512 Sep 28  2006 etc
drwxr-xr-x   2 other        512 Sep 28  2006 msgs
drwxr-xr-x  53 other       1024 Jun 30 09:52 pub
dr-xr-xr-x   5 other        512 Aug  9  2004 usr
drwxr-xr-x   2 other        512 Aug  9  2004 var
