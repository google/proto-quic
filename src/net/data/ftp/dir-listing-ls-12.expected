l
bin
-1
2005
9
1
0
0

d
dev
-1
2004
8
9
0
0

d
etc
-1
2006
9
28
0
0

d
msgs
-1
2006
9
28
0
0

d
pub
-1
1994
6
30
9
52

d
usr
-1
2004
8
9
0
0

d
var
-1
2004
8
9
0
0
